Create first GraphQL API specification (#9)
kyma-incubator/compass: components/gateway/README.md

# Gateway

> Gateway exposes GraphQL API.

Reviewer: ```suggestion
The Gateway exposes GraphQL API.
```

## Development

> After introducing changes in GraphQL schema, execute a `gqlgen.sh` script.

Reviewer: ```suggestion
After you introduce changes in the GraphQL schema, run the `gqlgen.sh` script.
```

> To run a Gateway, execute the following command:

Reviewer: ```suggestion
To run the Gateway, use the following command:
```

```
go run cmd/main.go
```

> GraphQL API playground is available at `localhost:3000` address.

Reviewer: ```suggestion
The GraphQL API playground is available at `localhost:3000`.
```
